pkg/client/client.go

```diff
@@ -222,30 +290,71 @@ func (c *Client) Read(ctx context.Context, query string) (string, error) {
 	return string(b), nil
 }
 
-// Write initiates a write query, returning the txn hash.
-func (c *Client) Write(ctx context.Context, query string) (string, error) {
-	req := &tableland.RelayWriteQueryRequest{Statement: query}
-	var res tableland.RelayWriteQueryResponse
+type writeConfig struct {
+	relay bool
+}
+
+// WriteOption controls the behavior of Write.
+type WriteOption func(*writeConfig)
 
-	if err := c.tblRPC.CallContext(ctx, &res, "tableland_relayWriteQuery", req); err != nil {
-		return "", fmt.Errorf("calling rpc relayWriteQuery: %v", err)
+// WriteRelay specifies whether or not to relay write queries through the Tableland validator.
+// Default behavior is false for main net EVM chains, true for all others.
+func WriteRelay(relay bool) WriteOption {
+	return func(wc *writeConfig) {
+		wc.relay = relay
 	}
+}
 
-	return res.Transaction.Hash, nil
+// Write initiates a write query, returning the txn hash.
+func (c *Client) Write(ctx context.Context, query string, opts ...WriteOption) (string, error) {
```

Comment:
`Write` can now be called with an option to relay writes. If that option isn't provided, the `Client` default is used.